docs/contributors/testing-overview.md

```diff
@@ -368,6 +368,10 @@ If you're using a different setup, you can provide the base URL, username and pa
 WP_BASE_URL=http://localhost:8888 WP_USERNAME=admin WP_PASSWORD=password npm run test-e2e
 ```
 
+### New Core Blocks
```

Comment:
There are also tests also on deprecations, so I'm thinking a more generic title might be better.

```suggestion
### Core Block Testing
```